TcpClient.Connect exception makes LISAv2 exit without running remaining tests.

https://github.com/LIS/LISAv2/blob/4c3c28212d287858dde3ffee2d28fb2b919555b1/Libraries/TestHelpers.psm1#L708-L726

```
07/25/2019 05:59:12 : [ERROR] EXCEPTION: Multiple ambiguous overloads found for "Connect" and the argument count: "2".
07/25/2019 05:59:12 : [ERROR] Source: Line 714 in script .\Libraries\TestHelpers.psm1.
```

This has happened when running Hyper-V P0 tests in sequence and LISAv2 exits without attempting to run the remaining tests, during NET-IFUP-IFDOWN test.

More Details:

```
07/10/2019 07:39:36 : [INFO ] echo '******' | sudo -S -s eval "export HOME=`pwd`;bash NET-IFUP-IFDOWN.sh > NET-IFUP-IFDOWN.log" is running in background with ID 705 ...
The RPC server is unavailable. (Exception from HRESULT: 0x800706BA)07/10/2019 07:39:36 : [WARN ] Get-IPv4ViaKVP: Unable to create Msvm_ComputerSystem object
The RPC server is unavailable. (Exception from HRESULT: 0x800706BA)07/10/2019 07:39:46 : [WARN ] Get-IPv4ViaKVP: Unable to create Msvm_ComputerSystem object
The RPC server is unavailable. (Exception from HRESULT: 0x800706BA)07/10/2019 07:39:56 : [WARN ] Get-IPv4ViaKVP: Unable to create Msvm_ComputerSystem object
07/10/2019 07:40:06 : [ERROR] Wait-ForVMToStartKVP: VM LISAv2-OneVM-ubuntuAzurebionic-YC72-636983133203-role-0 did not start KVP within timeout period (30)
07/10/2019 07:40:06 : [ERROR] Get-IPv4AndWaitForSSHStart: Unable to get ipv4 from VM LISAv2-OneVM-ubuntuAzurebionic-YC72-636983133203-role-0 via KVP within timeout period (30)
07/10/2019 07:40:06 : [INFO ] dos2unix: converting file C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to Unix format...
07/10/2019 07:40:06 : [INFO ] Uploading C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to lisa : False, port 22 using Password authentication
07/10/2019 07:40:11 : [WARN ] Error in upload, attempt 1, retrying
07/10/2019 07:40:22 : [INFO ] Uploading C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to lisa : False, port 22 using Password authentication
07/10/2019 07:40:27 : [WARN ] Error in upload, attempt 2, retrying
07/10/2019 07:40:37 : [INFO ] Uploading C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to lisa : False, port 22 using Password authentication
07/10/2019 07:40:42 : [WARN ] Error in upload, attempt 3, retrying
07/10/2019 07:40:52 : [INFO ] Uploading C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to lisa : False, port 22 using Password authentication
07/10/2019 07:40:57 : [WARN ] Error in upload, attempt 4, retrying
07/10/2019 07:41:08 : [INFO ] Uploading C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to lisa : False, port 22 using Password authentication
07/10/2019 07:41:13 : [WARN ] Error in upload, attempt 5, retrying
07/10/2019 07:41:23 : [INFO ] Uploading C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to lisa : False, port 22 using Password authentication
07/10/2019 07:41:28 : [WARN ] Error in upload, attempt 6, retrying
07/10/2019 07:41:38 : [INFO ] Uploading C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to lisa : False, port 22 using Password authentication
07/10/2019 07:41:44 : [WARN ] Error in upload, attempt 7, retrying
07/10/2019 07:41:54 : [INFO ] Uploading C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to lisa : False, port 22 using Password authentication
07/10/2019 07:41:59 : [WARN ] Error in upload, attempt 8, retrying
07/10/2019 07:42:09 : [INFO ] Uploading C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to lisa : False, port 22 using Password authentication
07/10/2019 07:42:14 : [WARN ] Error in upload, attempt 9, retrying
07/10/2019 07:42:24 : [INFO ] Uploading C:\LISAv2\YC72\TestResults\2019-09-07-23-55-12-0446\NET-IFUP-IFDOWN\runtest.sh to lisa : False, port 22 using Password authentication
07/10/2019 07:42:30 : [ERROR] EXCEPTION: Calling function - Upload-RemoteFile. Error in upload after 10 attempt, hence giving up
07/10/2019 07:42:30 : [ERROR] Source: Line 120 in script .\Libraries\TestHelpers.psm1.
07/10/2019 07:42:30 : [INFO ] Trying to connect to deployed VMs.
07/10/2019 07:42:30 : [ERROR] EXCEPTION: Multiple ambiguous overloads found for "Connect" and the argument count: "2".
07/10/2019 07:42:30 : [ERROR] Source: Line 714 in script .\Libraries\TestHelpers.psm1.
07/10/2019 07:42:30 : [INFO ] Copying all files back to original working directory: [****].
07/10/2019 07:42:41 : [INFO ] Setting workspace back to original location: [****]
07/10/2019 07:42:41 : [INFO ] Forcefully exiting with exit code 0 as ExitWithZero flag was set to True
07/10/2019 07:42:41 : [INFO ] LISAv2 exit code: 0
```

--- a/Libraries/TestHelpers.psm1
+++ b/Libraries/TestHelpers.psm1
@@ -705,7 +705,7 @@ Function Set-DistroSpecificVariables($detectedDistro)
 	}
 }
 
-Function Test-TCP($testIP, $testport)
+Function Test-TCP([string]$testIP, [Int32]$testport)
 {
 	$socket = New-Object Net.Sockets.TcpClient
 	$isConnected = "False"
@@ -713,7 +713,7 @@ Function Test-TCP($testIP, $testport)
 	{
 		$socket.Connect($testIP, $testPort)
 	}
-	catch [System.Net.Sockets.SocketException]
+	catch
 	{
 		Write-LogWarn "TCP test failed"
 	}

